src/Storefront/Resources/app/storefront/src/scss/page/checkout/_register.scss

```
@@ -19,7 +20,8 @@
 }
 
 @include media-breakpoint-up(lg) {
-    .is-act-checkoutregisterpage {
+    .is-act-checkoutregisterpage, // @deprecated tag:v6.8.0 - Will be removed, use active route class instead (see class selectors here).
```

We could write this more explicit, so there can be less misunderstandings what needs to be used in the future. Same for the other occurrences. 
```suggestion
    .is-act-checkoutregisterpage, // @deprecated tag:v6.8.0 - Body class `is-act-checkoutregisterpage` will be removed, use active route class `is-active-route-frontend-checkout-register-page` instead.
```